ui/page/channels/view.jsx

```diff
@@ -43,6 +45,15 @@ export default function ChannelsPage(props: Props) {
             header={__('Your Channels')}
             loading={fetchingChannels}
             uris={channels.map(channel => channel.permanent_url)}
+            headerAltControls={
+              <Button
+                iconSize={20}
+                label={__('New Channel')}
+                button="link"
+                icon={ICONS.EDIT}
```

Comment:
This icon shouldn't be used here. We don't have icons on other links in the header so we don't need one here.